packages/phone-number-privacy/signer/azure-templates/prometheus-template.json

```diff
@@ -0,0 +1,133 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+      "location": {
+          "type": "string"
+      },
+      "containerGroupName": {
+        "type": "string"
+      },
+      "containerName1": {
+          "type": "string"
+      },
+      "containerName2": {
+        "type": "string"
+      },
+      "imageType1": {
+          "type": "string",
+          "allowedValues": [
+              "Public",
+              "Private"
+          ]
+      },
+      "imageType2": {
+        "type": "string",
+        "allowedValues": [
+            "Public",
+            "Private"
+        ]
+      },
+      "imageName1": {
+          "type": "string"
+      },
+      "imageName2": {
+        "type": "string"
+      },
+      "osType": {
+          "type": "string",
+          "allowedValues": [
+              "Linux",
+              "Windows"
```

Comment:
Will the commands work on Windows as well?

Reply:
Hmm I think maybe just linux